examples/grpc-bidirectional-streaming/grpc_bidirectional_streaming.out

```diff
@@ -1,13 +1,14 @@
 # Create a Ballerina project and a module inside it.
-# Add grpc_bidirectional_streaming.bal file inside the module directory and
-# build the project.
-# For example, if you create a module named 'service', execute the following command to build
-# the 'service' module.
-$ ballerina build service
+# Generate mock service and stub code using grpc_bidirectional_streaming.proto file.
```

Comment:
Let's add file names within single back ticks. Apply elsewhere if applicable.

Reply:
Fixed in 71f8f2d